adoc/retail_chap_install.adoc

```diff
@@ -228,47 +231,48 @@ Here is a channel overview:
 . Run [command]``configure-proxy.sh``.
 +
 *TODO*: add more info about the last 3 actions if wanted; it's in Advanced Topics, Proxy QS.
+////
+
 
-After this basic proxy installation, use the formulas to deploy the Retail Branch Server functionality on it.
+For mass deployments, see xref:#retail.sect.admin.branch_mass_config[Branch Server Mass Configuration].
 
-For mass deployments, use the script included with the [package]``susemanager-retail-tools`` package.
 
 
-.Procedure: Assign and configure branch server formulas
+=== Procedure: Assign and configure branch server formulas
 
-Before you can configure the branch server formulas, you must have already configured networking, and know the subdomain name and IP address of the branch server.
+Before you configure the branch server formulas, ensure you have configured networking, and know the subdomain name and IP address of the branch server.
 
 [IMPORTANT]
 ====
 Check each formula to ensure the values are appropriate for your environment.
-Note that these instructions could vary depending on your network architecture.
+Note that these instructions will vary depending on your network architecture.
 ====
 
 . In the {susemgr} {webui}, open the details page for the branch server, and navigate to the [guimenu]``Formulas`` tab.
 . Check the [systemitem]``bind`` formula, and click [btn]``Save``.
 . Navigate to the menu:Formulas[Bind] tab, and set these parameters for Zone1:
-* In the [guimenu]``Name`` field, use the fully qualified domain name (FQDN) of the  branch server (for example: [systemitem]``branch1.mycompany.org``)
-* In the [guimenu]``Type`` field, select [systemitem]``master``
+* In the [guimenu]``Name`` field, use the fully qualified domain name (FQDN) of the  branch server (for example: [systemitem]``branch1.mycompany.org``).
```

Comment:
do not use mycompany.org, use example.org or example.com.  If more is needed, Novell once acquired a domain name for such a purpose.